### DISCUSSION TITLE
Collaboration suggest

### DISCUSSION THREAD

**Discussion by pemace** [garygru/yummyDSP#18] [Ideas]
Hello,

I'm a french control engineer and i program for fun. I'm interested in ESP32 programming and i've done an analog simulation synth.

Why not collaborate? I can program a Schroeder reverb node. Also i think that's interesting to do more waveshaping formulas.

Regards,
Pierre

**Comment by garygru** [comment]
Hi Pierre,
sure, that's why I open sourced it. Please add the nodes you miss. Looking forward to see it and I can help to integrate, when you face problems. I must admit that I currently don't have a lot of time for this project. Having someone keeping it alive is great! thanks,
Gary
